src/Moryx.Orders/Documents/WebDocument.cs

```diff
@@ -37,13 +40,15 @@ public WebDocument(string number, short revision, string url) : base(number, rev
         /// <inheritdoc />
         public override Stream GetStream()
         {
-            var request = WebRequest.Create(Url);
-            var response = request.GetResponse();
-
-            ContentType = response.ContentType;
-            var dataStream = response.GetResponseStream();
+            var getTask = Task.Run(() => _client.GetAsync(Url));
```

Comment:
Putting Async into task run combined with wait and result does not make a lot of sense. Why not just `_client.GetAsync(Url).Result`? Some effect...